programmers/위장.js

```diff
@@ -0,0 +1,15 @@
+function solution(clothes) {
+    const obj = clothes.reduce(function(cloth, kind) {
+        cloth[kind[1]] = cloth[kind[1]] ? cloth[kind[1]] + 1 : 1;
+        return cloth;
+    }, {});
+    
+    const value = Object.values(obj);
```

Comment:
```suggestion
    const clothesValues = Object.values(obj);
```
이것도 value 보다는 clothesValues